### PR TITLE
Logging site extension is missing net5.0 runtime store

### PR DESCRIPTION
**Pull request by shirhatti**

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -212,10 +212,10 @@
     <!-- Packages from 2.1, 3.1, and 5.0 branches used for site extension build. -->
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension21Version>2.1.1</MicrosoftAspNetCoreAzureAppServicesSiteExtension21Version>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension22Version>2.2.0</MicrosoftAspNetCoreAzureAppServicesSiteExtension22Version>
-    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31Version>3.1.17-servicing-21318-8</MicrosoftAspNetCoreAzureAppServicesSiteExtension31Version>
+    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31Version>3.1.18-servicing-21365-4</MicrosoftAspNetCoreAzureAppServicesSiteExtension31Version>
```
Comment (shirhatti) [inline eng/Versions.props:215]: @dougbu No idea if this is correct, I just grabbed the latest package version from the `dotnet3.1-internal-transport`/`dotnet-5-internal-transport` feeds

File: src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj [hunk 2]
```diff
@@ -17,7 +17,8 @@
     <NoSemVer20>true</NoSemVer20>
     <IsPackable>true</IsPackable>
     <IsShipping>true</IsShipping>
-    <IsShipping Condition=" '$(PreReleaseVersionLabel)' == 'preview' ">false</IsShipping>
+    <!-- TODO: Undo this change. I just wanted to test if the CI produces the right packages -->
+    <!-- <IsShipping Condition=" '$(PreReleaseVersionLabel)' == 'preview' ">false</IsShipping> -->
```
Comment (shirhatti) [inline src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj:21]: Will undo this change

File: eng/Versions.props (hunk 1 above)
Comment (dougbu) [reply eng/Versions.props:215]: Do _not_ change the properties in here. We want to stick w/ 3.1.17 until 3.1.18 is public and 5.0.8 until 5.0.9 is out.

File: src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj [hunk 3]
```diff
@@ -32,11 +33,8 @@
     <Reference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.2.1" PrivateAssets="All" />
     <Reference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x64" PrivateAssets="All" />
     <Reference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x86" PrivateAssets="All" />
-    <!--
-      Coming soon...
     <Reference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.5.0.x64" PrivateAssets="All" />
     <Reference Include="Microsoft.AspNetCore.AzureAppServices.SiteExtension.5.0.x86" PrivateAssets="All" />
-    -->
```
Comment (dougbu) [inline src/SiteExtensions/LoggingAggregate/src/Microsoft.AspNetCore.AzureAppServices.SiteExtension/Microsoft.AspNetCore.AzureAppServices.SiteExtension.csproj:39]: Good catch ❕

**Review by dougbu — COMMENTED** [review]
I'll approve once your testing is done, eng/Versions.props is restored, and you've undone your TODO bit

**Comment by shirhatti** [comment]
I just realized that the SiteExtension build step is skipped for PRs. @dotnet/aspnet-build how would I verify this change?

**Comment by dougbu** [comment]
> @dotnet/aspnet-build how would I verify this change?

Either
1. Add another temporary change to remove the `condition:` in .azure/pipelines/ci.yml for the 'Build SiteExtension' job
2. Push your branch to the dotnet-aspnetcore AzDO (mirror) repo and manually start a build there; don't create an internal PR

**Review by dougbu — APPROVED** [review]
Looks great if I squint and ignore the TODO items that you'll remove. Thanks again

**Comment by dougbu** [comment]
@JunTaoLuo are there current issues w/ Components.E2ETests and the SignalR tests❔ I can't see how @shirhatti's changes would cause the failures in the current build

**Comment by BrennanConroy** [comment]
> SignalR tests

Hmm, just added that test, I'll try to see if it's flaky and fix it

**Comment by shirhatti** [comment]
Verified that the right package got built
image

**Review by Pilchie — APPROVED** [review]

**Comment by BrennanConroy** [comment]
Am I confusing issues? In the email you said we didn't have the 6.0 folder with a picture, but this adds the 5.0 folder which was in the picture.

**Comment by shirhatti** [comment]
> Am I confusing issues? In the email you said we didn't have the 6.0 folder with a picture, but this adds the 5.0 folder which was in the picture.

I misspoke in the email. I was looking at the 5.0 package (since we don't publish SiteExtension packages for preview releases). Turns out I found another issue in the process 😅

**Review by BrennanConroy — APPROVED** [review]
